pom.xml

```diff
@@ -599,6 +599,7 @@
         <version.org.reactivestreams>1.0.4</version.org.reactivestreams>
         <version.org.syslog4j>0.9.30</version.org.syslog4j>
         <!-- Test dependency -->
+        <version.org.codehaus.plexus.plexus-utils>3.5.0</version.org.codehaus.plexus.plexus-utils>
```

Comment:
Please move this alphabetically with the others. The 'Test dependency' comment is unneeded IMHO. Documentation of why we have a particular dependency is better placed next to the dependencyManagement entry for it.

If you can also move the version.org.keycloak.... property into the correct alphabetical position I'd appreciate it.

Reply:
fixed, Thanks